lib/containers/common.pm

```diff
@@ -136,8 +137,10 @@ sub test_container_runtime {
 
 # Test a given image. Takes the image and container runtime (docker or podman) as arguments
 sub test_container_image {
-    my $image   = $_[0];
-    my $runtime = $_[1] //= "docker";
+    my $image = $_[0];
+    die "You must define the image!" unless $image;
```

Comment:
I personally like more "formal" error messages like "argument $image not provided" or "missing argument $image".

Also, since we are calling this function from another place, I prefer having everything inside @args, this way it is much clearer when you do the call to this function:
`test_container_image($image => $image_names->[$i], runtime => 'docker');`
instead of
`test_container_image($image_names->[$i], 'docker');`

Reply:
these comments apply to all appearances, so I'm skipping copy-pasting this in other places :)

Reply:
I'm with Jose on the formal error messages, but that's a cosmetic change

For the `test_container_image` - I can probably address this issue in another PR

Reply:
I also think we should tackle function signatures in a different PR, I think it makes sense to change more than one functions if we follow Jose's suggestion.

Reply:
@jlausuch I spoke with @ge0r and as I am already touching a lot of those functions I already did it. Please re-review 😉